index.html

```diff
@@ -278,7 +293,7 @@ <h3 class="text-contact">
         </ul>
         <p>rega1237@gmail.com</p>
         <div></div>
-      </section>
+      </footer>
     </main>
   </body>
```

Comment:
- [ ] From the section "project" in your code, Please make sure that the button `See Project` takes you to the GitHub repository. 

![fact](https://user-images.githubusercontent.com/63972670/160007263-9f6502bc-a35c-45e9-90f5-2ccc5ff617bc.JPG)